BaseFilter/sections/antiadblock.txt

```diff
@@ -850,6 +850,7 @@ starwarsaddicted.it,wuyong.fun,26g.me,myqqjd.com,poapan.xyz,javboys.com,idevicec
 ###chp_branding
 ##a[id][href="https://chpadblock.com/"][rel="noopener noreferrer"]
 ! specific
+@@||s-nr.dk/storage/js/arcads.js$domain=s-nr.dk

```

Comment:
I think that it would be better to add it to the same group as scriptlet.
Perhaps we could also modify these exceptions and add one rule like:
```adblock
@@/storage/js/*.js$~third-party,domain=wowmail.app|inipulsa.my.id|anon.tools|s-nr.dk
```

Reply:
ok i will move